src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -418,28 +448,157 @@ def _check_private_endpoint(cmd, registry_name, vnet_of_private_endpoint):  # py
     else:
         raise CLIError('DNS routing verification failed')
 
+# Validate --image input
+def validate_image(cmd,
```

Comment:
is this for a new command or a new argument? Either way please follow existing design pattern in the CLI

Reply:
@rosanch - will update to 'def _validate_image'. Is this what you are referring to?

Reply:
correct thanks